dogpile/cache/region.py

```diff
@@ -1397,6 +1400,16 @@ def delete_multi(self, keys: Sequence[KeyType]) -> None:
 
         self.backend.delete_multi(keys)
 
+    def get_age(self, key: KeyType) -> float | None:
+        """Get the elapsed time since a value was added to the cache in seconds.
+
+        If no value were added to the cache for this key, None is returned.
```

Comment:
probably this would be 1.2.3 since I'm still allergic to pure semver, so add this (maintain formatting):

```

.. versionadded:: 1.2.3  Added :meth:`.Region.get_age`
```

Reply:
[done](https://github.com/sqlalchemy/dogpile.cache/pull/247/commits/f39feb45825d63b88755de40877ee25345a6c8e7)